Exiled.API/Features/Room.cs

```diff
@@ -321,14 +328,14 @@ public void LockDown(float duration, DoorLockType lockType = DoorLockType.Regula
         /// <summary>
         /// Locks all the doors and turns off all lights in the room.
         /// </summary>
-        /// <param name="duration">Duration in seconds, or <c>-1</c> for permanent blackout.</param>
+        /// <param name="duration">Duration in seconds, or <c>-1</c> for permanent blackdown.</param>
         /// <param name="lockType">DoorLockType of the blackout.</param>
         /// <seealso cref="Map.TurnOffAllLights(float, ZoneType)"/>
         /// <seealso cref="Map.TurnOffAllLights(float, IEnumerable{ZoneType})"/>
-        public void Blackout(float duration, DoorLockType lockType = DoorLockType.Regular079)
+        public void BlackDown(float duration, DoorLockType lockType = DoorLockType.Regular079)
         {
             LockDown(duration, lockType);
-            TurnOffLights(duration);
+            Blackout(duration);
         }
 
```

Comment:
```suggestion
        /// <summary>
        /// Locks all the doors and turns off all lights in the room.
        /// </summary>
        /// <param name="duration">Duration in seconds, or <c>-1</c> for permanent blackout.</param>
        /// <param name="lockType">DoorLockType of the blackout.</param>
        /// <seealso cref="Map.TurnOffAllLights(float, ZoneType)"/>
        /// <seealso cref="Map.TurnOffAllLights(float, IEnumerable{ZoneType})"/>
        [Obsolete(Use BlackDown instead of this), false]
        public void Blackout(float duration, DoorLockType lockType = DoorLockType.Regular079) => BlackDown(duration, lockType)
        
```

Reply:
Verify after the commit merge if it's merge correctly